README.md

```diff
@@ -155,7 +155,7 @@ Linux kernel version 3-16.0-rc5)
 
 ```
 $ sudo apt-get install zlib1g-dev libglib2.0-dev libpixman-1-dev libfdt-dev \
-		       libc6:i386 libstdc++6:i386 libz1:i386 cscope
+		       libc6:i386 libstdc++6:i386 libz1:i386 cscope netcat
```

Comment:
`SY wrote: Should I add an extra section in 4.4 QEMU to describe we need to install netcat ?`
I would say, no, not needed. After I've been at the GP conference I think we need to clean-up, update and re-organize our instructions.

`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`